lib/takeaway.rb

```diff
@@ -0,0 +1,16 @@
+# class Takeaway
+
+# attr_reader :menu, :selection_of_dishes
+
+#     @menu = {
+#         "lasagne" => 12,
+#         "oysters" => 14,
+#         "fries" => 4,
+#         "green_salad" => 9,
+#         "rump_steak" => 20,
+#         "tiramisu" => 25
+#     }
+    
+#     @selection_of_dishes = []
+
+# end
```

Comment:
Good idea to keep the menu as a separate class, this would definitely allow for more functionality than hard coding it in the initialize method

Reply:
Yeah, you're totally right. Thanks for all these super useful comments!